### PR TITLE
BUGHERD FIXUPS

### PR DESCRIPTION
**Pull request by josephgknox**
# READY FOR REVIEW

# Summary
- Added accent dash to Featured Publications listing on person nodes to mirror mockups
- Tightened up spacing for exposed filters
- Updated secondary button styles to mirror mockup
- Un-hid yellow accent color when hovering on main menu
- Updated homepage featured publications view
- Added band-aid fix for more links when no title is present on a horizontal card (discussion to follow for future enhancement; left my comments/suggestions on ticket)
- Reduced the depth of main menu to 2
- Updated displaying of heading dashes in necessary places

# Needed By (Date)
- End of sprint (9/13)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test to ensure that aforementioned FIXUPS have been addressed and are propagating (I can help provide examples if needed)

# Affected Projects or Products
- suhumsci (theme)
- Archaeology

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-163
- https://stanfordits.atlassian.net/browse/ARCH-347
- https://stanfordits.atlassian.net/browse/ARCH-352
- https://stanfordits.atlassian.net/browse/ARCH-359
- https://stanfordits.atlassian.net/browse/ARCH-370
- https://stanfordits.atlassian.net/browse/ARCH-377
- https://stanfordits.atlassian.net/browse/ARCH-381
- https://stanfordits.atlassian.net/browse/HSD8-266

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/themes/humsci/su_humsci_theme/patterns/molecules/vertical-card/vertical-card.html.twig [hunk 1]
```diff
@@ -17,6 +17,21 @@
         {{ description }}
       </div>
     {% endif %}
+    {% if author|render|trim or year|render|trim %}
```
Comment (pookmish) [inline docroot/themes/humsci/su_humsci_theme/patterns/molecules/vertical-card/vertical-card.html.twig]: probably dont need the `|render|trim` part. theres functionality occuring elsewhere that cleans up items that have no rendering.

File: docroot/themes/humsci/su_humsci_theme/scss/components/molecules/_hover-menu.scss [hunk 2]
```diff
@@ -206,6 +206,7 @@
         .ui-widget {
           background: $color-white;
 
+          @include margin(1rem null null);
```
Comment (pookmish) [inline docroot/themes/humsci/su_humsci_theme/scss/components/molecules/_hover-menu.scss]: This changes the look when a top menu item is a real link. see screenshots: 
screen shot 2018-09-05 at 11 41 48 am
screen shot 2018-09-05 at 11 41 58 am

File: config/default/core.entity_view_display.node.hs_publications.hs_vertical_card.yml [hunk 3]
```diff
@@ -54,14 +56,19 @@ bundle: hs_publications
 mode: hs_vertical_card
 content:
   field_hs_publication_author:
-    type: entity_reference_entity_view
+    type: entity_reference_label
```
Comment (pookmish) [inline config/default/core.entity_view_display.node.hs_publications.hs_vertical_card.yml]: would we not want the user to be able to click on the author's name to view all publications by that author?

Comment (josephgknox) [reply config/default/core.entity_view_display.node.hs_publications.hs_vertical_card.yml]: Good question. I don't think so, not in this case, as it wasn't scoped or mocked. If we did do it (here or in the future), we'd likely want to be sure we did it everywhere, not just on the homepage card. Let's bring this up as a future enhancement later, as it hasn't been asked for.

**Pull request merged**